lib/main_common.pm

```diff
@@ -369,6 +369,9 @@ sub load_boot_tests {
         set_var("DELAYED_START", "1");
         loadtest "autoyast/pxe_boot";
     }
+    elsif (get_var('BACKEND', 'spvm')) {
```

Comment:
Should not be `check_var('BACKEND', 'spvm')` instead? Otherwise `bootloader_spvm` seems to be always executed (and fails of course!).

Reply:
fixed